fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -2205,6 +2206,101 @@ countries:
                 issue_months: [3, 4, 5, 6]
                 start_year: 1996
 
+
+    guatemala-may:
+        logo: Guatemala_IRI_120x48.png
+        resolution: [.1, .1]
+        center: [-89, 16]
+        zoom: 8
+        rotation: 0
+        marker:  [-90, 16]
+        origin: [0, 0]
+        shapes:
+            - name: National
+              sql: select adm0_code as key, adm0_name as label, ST_AsBinary(the_geom) as the_geom
+                  from iridb.g2015_2012_0 where adm0_name = 'Guatemala'
+              vuln_sql: select adm0_code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Guatemala' group by 1, 2
+
+            - name: Department
+              sql: select (adm0_code, adm1_code) as key, adm1_name as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.g2015_2014_1
+                  where adm0_name = 'Guatemala'
+              vuln_sql: select (adm0_code, adm1_code) as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Guatemala' group by 1, 2
+
+            - name: Municipality
+              sql: select (adm0_code, adm1_code, adm2_code) as key, adm2_name as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.g2015_2014_2
+                  where adm0_name = 'Guatemala'
+              vuln_sql: select (adm0_code, adm1_code, adm2_code) as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Guatemala' group by 1, 2
+
+        datasets:
+            defaults:
+                predictors:
+                    - pnep-v1
+                    - rain
+                predictand: bad-years
+            observations:
+                rain:
+                    label: Rain
+                    description: CHIRPS average precipitation over May
+                    path: guatemala/rain-may.zarr
+                    var_names:
+                        value: prcp_est
+                        lat: Y
+                        lon: X
+                        time: T
+                    colormap: precip
+                    lower_is_worse: yes
+                enso_state: *enso_3mo
+                bad-years:
+                    label: Reported bad years
+                    path: guatemala/bad-years-aso-dummy.zarr
+                    var_names:
+                        value: bad
+                        time: T
+                    colormap: precip
+                    lower_is_worse: no
+                    format: bad
+                vhi:
+                    label: VHI
+                    description: May Vegetation Health Index from NESDIS STAR
+                    path: guatemala/vhi-may.zarr
```

Comment:
This file doesn't exist yet?

Reply:
sorry, not this file. it's the next one: guatemala/prcp-may-v1.zarr

Reply:
We just tried to create it (ran the fbf-update-data script), but it may not have gone through -- is there another step we need to do?